content/cheatsheets/mysql.mdx

```diff
@@ -16,24 +16,98 @@ published: true
 ```
 
 ```sql title="CREATE"
+    -- create table
     CREATE TABLE TableName (
```

Comment:
Could you help to modify this line to: CREATE TABLE OR REPLACE <TableName>

Reply:
Miss regina san. CREATE TABLE OR REPLACE is not exist for TABLE. Need to drop the table first then re-create 👯